openslides/agenda/static/templates/agenda/item-detail.html

```diff
@@ -34,51 +34,109 @@ <h3 os-perm="agenda.can_manage" translate>Comment</h3>
 
 
 <h3 translate>List of speakers
-  <span ng-if="item.speakerListClosed" class="label label-danger" translate>closed</span>
   <span os-perms="agenda.can_manage">
     <button ng-if="item.speakerListClosed" ng-click="closeList(false)"
-        class="btn btn-sm btn-default" translate>
-      Open list
+        class="btn btn-sm btn-danger" translate>
+      <b>Closed</b>
     </button>
     <button ng-if="!item.speakerListClosed" ng-click="closeList(true)"
-        class="btn btn-sm btn-default" translate>
-      Close list
+        class="btn btn-sm btn-success" translate>
+      <b>Opened</b>
     </button>
   </span>
+  <!-- project list -->
+  <a os-perms="core.can_manage_projector" class="btn btn-default btn-sm"
+      ng-class="{ 'btn-primary': item.isProjected() }"
+      ng-click="item.project()"
+      title="{{ 'Project item' | translate }}">
+    <i class="fa fa-video-camera"></i> Project list
+  </a>
 </h3>
 
 <!-- TODO:
    * project list
-   * show old/current/next speakers
-   * start/stop speech
-   * button 'put/remove me on/from the list'
-   * check permissions
+   * show only 'add me' OR 'remove me' button
 -->
-<ol>
-  <li ng-repeat="speaker in item.speakers">
+<div class="well">
+  <p>
+  <b translate>Old speakers:</b>
```

Comment:
The omitted closing p-tag is ended after  the button. So you should add white-spaces before the button.